### ISSUE TITLE
Response type not showing for array of types containing Union property

### ISSUE THREAD

**Issue by dc950** [mulesoft/api-console#578]
If I have the following RAML:

```
#%RAML 1.0
title: Union Type Error
version: 1.0.0
baseUri: /union-type-error
protocols: [ HTTPS ]

/event:
  get:
    responses:
      200:
        body:
          application/json:
            type: event
            example:
              eventName: "Christmas Party"
              date: Christmas
/events:
  get:
    responses:
      200:
        body:
          application/json:
            type: event[]
            example:
              - eventName: "Christmas Party"
                date: Christmas
              - eventName: "Bob's Birthday Party"
                date: 2018-10-12

types:
  event:
    properties:
      eventName: string
      date: eventDate
  specialDate:
    type: string
    enum: [Christmas, Easter]
  eventDate: specialDate | date-only
```

The Response Type view for the `events` method is just displayed as a set with an empty object:

![console-union-blank](https://user-images.githubusercontent.com/9059291/43394554-0cdff6b0-93f3-11e8-868e-474b8b5edb85.png)

It works fine, however, for `event` which uses the same type but not as an array.

**Comment by jarrodek** [comment]
Hi,
Thank you for issue report. This was addressed in API console version 5. I will investigate if this is something that can be fixed in current version as it may require serious changes in the code.

**Comment by jarrodek** [comment]
This is fixed in latest release of APIC 5.0.0
